### PR TITLE
Make cleanup pass on `CheckModifiersDialog` application

### PR DESCRIPTION
**Pull request by stwlam**

### CODE REVIEW

File: src/module/system/check-modifiers-dialog.ts [hunk 1]
```diff
@@ -72,10 +81,19 @@ export class CheckModifiersDialog extends Application {
...
+                if (options instanceof Set) {
+                    options.delete(option);
+                } else {
+                    options?.findSplice((o) => o === option);
+                }
+            } else if (options instanceof Set) {
+                options.add(option);
             } else {
-                this.context.options?.push(`substitute:${substitution.slug}`);
+                options?.push(`substitute:${substitution.slug}`);
```
Comment (CarlosFdez) [inline src/module/system/check-modifiers-dialog.ts:96]:
```suggestion
                options?.push(option);
```

**Comment by CarlosFdez** [comment]
I think for a proper cleanup path you should probably lock in context.options's type earlier (like in the constructor?), that way you don't need to resolve set vs array in getData()

**Comment by stwlam** [comment]
It's temporary until the next PR.

**Comment by CarlosFdez** [comment]
Then sure, a little regression is ok then. Was wondering because as is this isn't actually a cleanup.

**Pull request merged**